backend/de.metas.business/src/main/java/de/metas/costrevaluation/impl/CostRevaluationLine.java

```diff
@@ -0,0 +1,73 @@
+package de.metas.costrevaluation.impl;
+
+import de.metas.costing.CostPrice;
+import de.metas.product.ProductId;
+import de.metas.quantity.Quantity;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.annotation.Nullable;
+
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2018 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+@Getter
+@ToString
```

Comment:
Since I see you're not doing anything fancy in the constructor, why not ditch it and just annotate the class with
```suggestion
@Value
@Builder
```
?